automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/security/AuthDomainSpec.scala

```diff
@@ -26,171 +28,460 @@ class AuthDomainSpec extends FreeSpec /*with ParallelTestExecution*/ with Matche
+              withCleanUp {
+                val listPage = signIn(Config.Users.george)
+                val summaryPage = listPage.openWorkspaceDetails(projectName, workspaceName)
+
+                val cloneWorkspaceName = workspaceName + "_clone"
+                val cloneModal = summaryPage.ui.clickCloneButton()
+                cloneModal.ui.readLockedAuthDomainGroups() should contain(groupOneName)
+                cloneModal.ui.readLockedAuthDomainGroups() should contain(groupTwoName)
+
+                register cleanUp {
+                  api.workspaces.delete(projectName, cloneWorkspaceName)(AuthTokens.george)
+                }
+                cloneModal.cloneWorkspace(projectName, cloneWorkspaceName)
+                cloneModal.cloneWorkspaceWait()
+                summaryPage.cloneWorkspaceWait()
+
+                val cloneSummaryPage = new WorkspaceSummaryPage(projectName, cloneWorkspaceName).awaitLoaded()
+                cloneSummaryPage.ui.readWorkspaceName should be(cloneWorkspaceName)
+                cloneSummaryPage.ui.readAuthDomainGroups should include(groupOneName)
+                cloneSummaryPage.ui.readAuthDomainGroups should include(groupTwoName)
+              }
+            }
           }
         }
       }
-
-      "should be visible and accessible when shared with a group" in withWebDriver { implicit driver =>
-        withGroup("AuthDomainSpec", List(Config.Users.george.email)) { authDomainName =>
-          withWorkspace(projectName, "AuthDomainSpec_share", Some(authDomainName), List(AclEntry(s"GROUP_$authDomainName@quality.firecloud.org", WorkspaceAccessLevel.Reader))) { workspaceName =>
-            val listPage = signIn(Config.Users.george)
-            listPage.filter(workspaceName)
-            listPage.ui.looksRestricted(projectName, workspaceName) shouldEqual true
-
-            val summaryPage = listPage.openWorkspaceDetails(projectName, workspaceName).awaitLoaded()
-            summaryPage.ui.readAuthDomainGroups should include (authDomainName)
+      "can be cloned and have a group added to the auth domain" in withWebDriver { implicit driver =>
+        withGroup("AuthDomainSpec", List(Config.Users.george.email)) { groupOneName =>
+          withGroup("AuthDomainSpec", List(Config.Users.george.email)) { groupTwoName =>
+            withGroup("AuthDomainSpec", List(Config.Users.george.email)) { groupThreeName =>
+              withWorkspace(projectName, "AuthDomainSpec_share", Set(groupOneName, groupTwoName), List(AclEntry(Config.Users.george.email, WorkspaceAccessLevel.Reader))) { workspaceName =>
+                withCleanUp {
+                  val listPage = signIn(Config.Users.george)
+                  val summaryPage = listPage.openWorkspaceDetails(projectName, workspaceName)
+
+                  val cloneWorkspaceName = workspaceName + "_clone"
+                  val cloneModal = summaryPage.ui.clickCloneButton()
```

Comment:
Since another test already checked that the groups cannot be removed when cloning, this test could simply use `WorkspaceSummaryPage.cloneWorkspace` instead of driving the modal itself. Not a big deal either way but you could simplify this if you want to. Just make sure to only pass the new group in since the existing groups will be preselected.